consensus/consensus-types/src/sync_info.rs

```diff
@@ -19,6 +22,10 @@ pub struct SyncInfo {
     highest_commit_cert: QuorumCert,
     /// Optional highest timeout certificate if available.
     highest_2chain_timeout_cert: Option<TwoChainTimeoutCertificate>,
+    /// Highest ordered decision known to the peer.
+    highest_ordered_decision: Option<LedgerInfoWithSignatures>,
+    /// Highest commit decision known to the peer.
+    highest_commit_decision: Option<LedgerInfoWithSignatures>,
```

Comment:
You can annotate them with `#[serde(default)]` to ensure backwards compatibility?

Reply:
To test whether `#[serde(default)]` would work out, I tried this unit test. It raised error. So, `#[serde(default)]` may not work.

```
    #[derive(Deserialize, Serialize, Clone, Eq, Debug, PartialEq)]
    struct SyncInfo1 {
        a: u64,
        b: u64,
    }

    #[derive(Deserialize, Serialize, Clone, Eq, Debug, PartialEq)]
    struct SyncInfo2 {
        a: u64,
        b: u64,
        #[serde(default)]
        c: Option<u64>,
    }
    
    #[test]
    fn bcs_test() {
        let value = SyncInfo1 { a: 1, b: 2 };
        let serialized =  bcs::to_bytes_with_limit(&value, 200).map_err(|e| anyhow!("{:?}", e));
        println!("{:?}", serialized);
        let deserialized = bcs::from_bytes_with_limit::<SyncInfo2>(&serialized.unwrap(), 200).map_err(|e| anyhow!("{:?}", e));
        println!("{:?}", deserialized);
    }
```